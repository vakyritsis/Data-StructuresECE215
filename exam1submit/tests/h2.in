155  99  56  29  25  11  55  73  65  83 137 122 139 138 142 318 220 198 192 168 194 205 203 208 263 245 230 259 279 273 303 422 362 347 332 354 394 389 400 469 449 448 465 490 471 500 -1
-h
