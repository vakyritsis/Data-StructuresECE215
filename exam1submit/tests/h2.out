155#6 99#5 56#4 29#3 25#2 11#1 55#1 73#2 65#1 83#1 137#3 122#1 139#2 138#1 142#1 318#5 220#4 198#3 192#2 168#1 194#1 205#2 203#1 208#1 263#3 245#2 230#1 259#1 279#2 273#1 303#1 422#4 362#3 347#2 332#1 354#1 394#2 389#1 400#1 469#3 449#2 448#1 465#1 490#2 471#1 500#1 
